products/firefox_android/localize_android.md

```diff
@@ -0,0 +1,45 @@
+# Firefox for Android - How to localize
+
+Please make sure to go through the [Firefox for Android README](/products/firefox_android/README.md) first to get a clear overview of what this project covers.
+
+The release cycle for Firefox for Android is the same as Firefox for Desktop, which is generally 6 weeks - but can sometimes be 8 weeks. "Merge day" is when localizations move to the next train - and the last day available to include localization updates to the next Beta version if you work on tools like Pootle.
+
+You can see the [detailed schedule for each release here](https://wiki.mozilla.org/RapidRelease/Calendar).
+
+Note: please make sure that you've already localized mozilla.org before you start working on Firefox for Android or any other projects.
+
+## The importance of regularly checking your dashboard
+
+Please keep in mind during your entire localization process to regularly check [your locale's dashboard](https://l10n.mozilla.org/shipping/dashboard): this is extremely important since if you break something, and your locale is shipping on multi-locale builds (so, the builds that actually ship on the Play Store), you will need to fix it as soon as possible since it breaks the build for everyone!
+
+Checking your dashboard regularly should be something you should start doing from the very beginning when you start localizing a project, so that it becomes a habit. This helps reduce the chance of breaking the builds in the future, once your locale is actually shipping, and generally helps catch errors earlier on. It also gives you an overview of your locale's health and status.
+
+When you are in your dashboard, and you notice that there is a red tick in your locale's "Action" column, this means there is a critical error that you need to fix. Clicking on your locale's "Status" column will give you an indication on what needs to be fixed, and where to find the error.
```

Comment:
Actually, warnings are not even displayed on that page (didn't remember that :-\)
https://l10n.mozilla.org/teams/ltg

I meant that they should not be focusing on orange or green icon, only on the number of errors and warnings (in /shipping). On the team's page, only on error notices and finding them in the linked page.